src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -101,6 +116,18 @@ public bool GetRandomBoolean(double probability = 0.5)
             return _random.NextDouble() < probability;
         }
 
+        public void PopulateWithRandomHeaders(HttpRequestHeaders headers)
+        {
+            int numHeaders = _random.Next(maxValue: 100);
```

Comment:
`maxValue` isn't named elsewhere, can we update for consistency?

Reply:
Addressed